tests/e2e/upgrade/admin-user.wdio-spec.js

```diff
@@ -1,34 +0,0 @@
-const utils = require('@utils');
-const loginPage = require('@page-objects/default/login/login.wdio.page');
-
-describe('admin users', () => {
-  it('should allow to update the admin password and login successfully', async () => {
-    const newPassword = 'medic.456';
-    const adminUser = {
-      username: 'admin2',
-      password: 'medic.123',
-      roles: [ 'admin' ],
-      contact: { name: 'Philip' },
-      place: { name: 'place', type: 'district_hospital' },
-    };
-    await utils.createUsers([ adminUser ]);
-
-    const membership = await utils.request({ path: '/_membership' });
-    const nodes = membership.all_nodes;
-    for (const nodeName of nodes) {
-      await utils.request({
-        method: 'PUT',
-        path: `/_node/${nodeName}/_config/admins/${adminUser.username}`,
-        body: `"${adminUser.password}"`,
-      });
-    }
-
-    await utils.request({
-      path: `/api/v1/users/${adminUser.username}`,
-      method: 'POST',
-      body: { password: newPassword }
-    });
-
-    await loginPage.login({ username: adminUser.username, password: newPassword, adminApp: true });
-  });
-});
```

Comment:
@dianabarsan This test is literally trying to change the admin password which is no longer possible as of this PR. The functionality being testing is a duplicate of an integration test we have so I'm not clear on why we test it in the upgrade suite too. Do you remember? Was it actually testing the password change or was it also doing something else?

Reply:
Thanks @garethbowen . I think it served as a check that new admins can still get created and login. I think keeping the test that just creates the new admin user and allows them to login to the admin app should be ok?

Reply:
Thanks! Added it back and it passed. Too easy.